sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -27,9 +27,7 @@
 import static com.microsoft.appcenter.http.DefaultHttpClient.METHOD_GET;
 import static com.microsoft.appcenter.http.DefaultHttpClient.METHOD_POST;
 import static com.microsoft.appcenter.http.HttpUtils.createHttpClient;
-import static com.microsoft.appcenter.storage.Constants.LOG_TAG;
-import static com.microsoft.appcenter.storage.Constants.SERVICE_NAME;
-import static com.microsoft.appcenter.storage.Constants.STORAGE_GROUP;
+import static com.microsoft.appcenter.storage.Constants.*;
```

Comment:
Please don't do that and always commit with via Android Studio with the following options:

<img width="282" alt="screen shot 2019-02-28 at 4 51 14 pm" src="https://user-images.githubusercontent.com/623672/53608820-16bbd180-3b79-11e9-8cbb-17c2eefeeaf3.png">